### ISSUE TITLE
Support running tests without wallaby

### ISSUE THREAD

**Issue by pavsaund** [dolittle/JavaScript.Build#21]
Wallaby test runner is a paid variant and is not available to everyone. We should continue to support running tests without the Wallaby.js runner.

**Comment by smithmx** [comment]
I don't think we should spend any time on this.  We should actually be much more prescriptive at this time.  Use this.  Do that.  Every extension or substitution that we allow is something we have to support, develop, test.  I would rather we had a "how to" that gives some pointers in how to replace wallaby with a free variant than did it.
